server/sonar-web/src/main/js/apps/issues/components/BulkChangeModal.js

```diff
@@ -65,16 +68,17 @@ type State = {|
   transition?: string,
   type?: string
 |};
+*/
 
-const hasAction = (action: string) => (issue: Issue): boolean =>
+const hasAction = (action /*: string */) => (issue /*: Issue */) =>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)